elements/AAC-850m6/update.js

```diff
@@ -598,16 +601,16 @@ var update = function(instance, properties, context) {
       $(`#${instance.data.id}`).children().eq(3).hide();
     });
       
+    quill = instance.data.quill;
```

Comment:
This part is basically changing the order so that reference to 'quill' exists first before trying to access it further in 'quill.root'

Reply:
haha was about to do the same